extensions/_modules/vault.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+"""
+This module provides methods for interacting with Hashicorp Vault via the HVAC
+library.
+"""
+from __future__ import absolute_import
+
+import logging
+import inspect
+from functools import wraps
+
+log = logging.getLogger(__name__)
+EXCLUDED_HVAC_FUNCTIONS = ['initialize']
+
+try:
+    import hvac
+    import requests
+    DEPS_INSTALLED = True
+except ImportError:
+    log.debug('Unable to import the HVAC library.')
+    DEPS_INSTALLED = False
+
+__all__ = ['initialize', 'is_initialized']
+
+class InsufficientParameters(Exception):
+    pass
+
+def __virtual__():
+    return DEPS_INSTALLED
+
+
+def _build_client(url='https://localhost:8200', token=None, cert=None,
+                  verify=True, timeout=30, proxies=None, allow_redirects=True,
+                  session=None):
+    client_kwargs = locals()
```

Comment:
This seems ok. Locals can never include anything besides the kwargs here, right?